### DISCUSSION TITLE
Property 'amount' is missing in type '{}' but required in type 'MyValueResponse'.ts(2741)

### DISCUSSION THREAD

**Discussion by imoisharma** [node-fetch/node-fetch#1470] [Q&A]
Can anyone help me with this, as I am pretty new with nodejs

**Issue**
Hi there, my code was working fine, then I upgrade the node-fetch version from v2->v3 after receiving the npm audit warning. And, now it complaining about `Property 'amount' is missing in type '{}' but required in type 'MyValueResponse'.ts(2741)`

The amount is declared under another file and here it's source code:
```ts
export interface MyValueResponse {
  amount: number[];
}
```

**Comment by jimmywarting** [comment]
Need more context (code example)

I assume you do something like: `response.json()` somewhere and it returns a `Promise<unknown>` or something?
you have to convert the type to `MyValueResponse` somehow.
